src/basic-syntax/functions-interlude.md

```diff
@@ -21,3 +21,11 @@ fn main() {
     println!("cash prize: {}", pick_one(500, 1000));
 }
 ```
+
+<details>
+
+* When using generics, the standard library's `Into<T>` can provide a kind of limited
+  polymorphism on argument types. We will see more details in exercise 7.1.
```

Comment:
Referencing section numbers is dangerous: they will go out of sync very easily. Please just say "later today" or "tomorrow" or similar.

Reply:
Excellent point. Amended to "later section".